modules/op-creating-and-verifying-task-run-signatures-without-any-additional-authentication.adoc

```diff
@@ -76,15 +73,13 @@ NAME                            STATUS
 ----
 $ export TASKRUN_UID=$(tkn tr describe --last -o  jsonpath='{.metadata.uid}')
 $ tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/signature-taskrun-$TASKRUN_UID}" > signature
```

Comment:
@mramendi Really sorry for asking for last change and also for late reply

@ppitonak  and I did some verification around this and got into the final conclusion that we need to update the following change from 

```bash=
export TASKRUN_UID=$(tkn tr describe --last -o jsonpath='{.metadata.uid}')
tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/signature-taskrun-$TASKRUN_UID}" > signature
tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/payload-taskrun-$TASKRUN_UID}" | base64 -d > payload
```

to 

```bash
export TASKRUN_UID=$(tkn tr describe --last -o  jsonpath='{.metadata.uid}')
tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/signature-taskrun-$TASKRUN_UID}" | base64 -d > sig
```

Reply:
cc @ppitonak